stubs/aiofiles/aiofiles/os.pyi

```diff
@@ -4,10 +4,12 @@ from asyncio.events import AbstractEventLoop
 from os import stat_result
 from typing import Any, Sequence, Union, overload
 
+from . import ospath as path
+
 _FdOrAnyPath = Union[int, StrOrBytesPath]
 
 async def stat(
-    path: _FdOrAnyPath,
+    path: _FdOrAnyPath,  # noqa: F811
```

Comment:
The `noqa`s are due to importing `ospath as path` and then `path` also being used for several arguments.